grc/utils/form_custom_validators.py

```diff
@@ -203,3 +203,40 @@ def file_length_check(form, field):
                 raise ValidationError(f'The selected file must be smaller than {max_size_in_mb}MB')
 
     return file_length_check
+
+
+def fileVirusScan(form, field):
+    if current_app.config['AV_API'] is None:
+        return
+    if (field.name not in request.files or request.files[field.name].filename == ''):
+        return
+
+    print('Scanning %s' % current_app.config['AV_API'], flush=True)
+    from pyclamd import ClamdNetworkSocket
```

Comment:
Should this import go at the top of the file?

Reply:
As this validator is rarely used it's better in the function